data/856/331/59/85633159.geojson

```diff
@@ -4,8 +4,16 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "eurostat:country_code":"UK",
+    "eurostat:level_code":"0",
+    "eurostat:name_latin":"United Kingdom",
+    "eurostat:name_local":"United Kingdom",
```

Comment:
Probably not for this PR set, but would be harvest these names for new/updated `name:` properties.

Reply:
Filed https://github.com/whosonfirst-data/whosonfirst-data/issues/2139